docs/gettingStarted/ANGULAR.md

```diff
@@ -1 +1,242 @@
+
+To install Zeplin CLI from npm, run the following command on your Terminal/Command Prompt:
+
+```sh
+npm install -g @zeplin/cli
+```
+
+Alternatively, if you're using npm in your project, you can add `@zeplin/cli` as a `devDependency`.
+
+## 3. Install CLI Angular plugin
+
+Zeplin CLI uses plugins to generate documentation, snippets and links from components—check out our [list of plugins](README.md#Plugins).
+
+Since we're using Angular, we'll install the official Angular plugin from npm, by running the following command:
+
+```sh
+npm install -g @zeplin/cli-connect-angular-plugin
+```
+
+Now, we'll update our configuration file to use the plugin. We can do that by adding it under the `plugins` list, like so:
+
+```json
+{
+    "plugins": [
+        {
+            "name": "@zeplin/cli-connect-angular-plugin"
+        }
+    ],
+    "projects": [],
+    "styleguides": [
+        "5cd486b18a64c1414be004fb"
+    ],
+    "components": [
+        {
+            "path": "src/components/Button/Button.ts",
+            "zeplinNames": [
+                "Controls / Button / Primary"
+            ]
+        }
+    ]
+}
+```
+
+Next up, we'll run the CLI tool!
+
+## 4. Run Zeplin CLI
+
+It's time! Let's **run the CLI tool and see Connected Components in action** within Zeplin. 🎉
+
+Run the following command—if it's your first time, you'll need to login to Zeplin first:
+
+```sh
+zeplin connect
+```
+
+Now head back to Zeplin and click on one of the components you connected. You should see an output similar to this:
+
+<img src="../../img/zeplinConnectedComponent-angular.png" alt="Connected component in Zeplin" width="600" />
+
+## 5. Add links _(Optional)_
+
+Connected Components also lets you add links to various sources like your Storybook, repository, wiki and so on. In the screenshot above, notice that we have links to GitHub and Storybook.
+
+To add links to your components, check out these guides:
+
+- [Adding Storybook links](../link/STORYBOOK.md)
+- [Adding Styleguidist links](../link/STYLEGUIDIST.md)
+- [Adding repository links](../link/REPOSITORY.md), e.g. GitHub, GitLab, Bitbucket
+- [Adding custom links](../link/CUSTOM.md), e.g. internal Design System wiki
+
+## 6. More code snippets _(Optional)_
+
+By default, Angular plugin generates a code snippet using only one of the possible selectors of the Angular component. In most cases the generated code snippet will be enough to get a glimpse of how it can be use, but if you want to see all possible combinations, you can use following configuration to do so.
+
+```json
+…
+    "plugins" : [{
+        "name": "@zeplin/cli-connect-angular-plugin",
+        "config": {
+            "useFullSnippet": true,
+            "useFullDescription": true,
+        }
+    }],
+…
+```
+
+The output will be something like the below:
+
+<img src="../../img/zeplinConnectedComponent-angular-full.png" alt="Connected component in Zeplin" width="600" />
```

Comment:
File is missing for now. I've created an example Angular component looking like React example, will how it is looking like asap.